plugin.yaml

```diff
@@ -452,6 +452,8 @@ node_types:
     derived_from: cloudify.nodes.Root
     properties:
       terminal_auth:
+        description: >
```

Comment:
Can we give a little more info about what terminal credentials are supposed to be specifically?

Reply:
fixed